data/events.yaml

```diff
@@ -39,3 +39,75 @@ x-events:
   properties:
     cluster_id: UUID
     host_id: UUID
+
+- name: cluster_registration_failed
+  message: "Failed to register cluster with name '{cluster_name}'. Error: {error}"
+  event_type: cluster
+  severity: "error"
+  properties:
+    cluster_id: UUID
+    cluster_name: string
+    error: string
+
+- name: registered_cluster
+  message: "Registered cluster {cluster_name}"
+  event_type: cluster
+  severity: "info"
+  properties:
+    cluster_id: UUID
+    cluster_name: string
+
+- name: add_hosts_cluster_registration_failed
+  message: "Failed to register add-hosts cluster with name '{cluster_name}'. Error: {error}"
+  event_type: cluster
+  severity: "error"
+  properties:
+    cluster_id: UUID
+    cluster_name: string
+    error: string
+
+- name: registered_add_hosts_cluster
+  message: "Registered add-hosts cluster {cluster_name}"
+  event_type: cluster
+  severity: "info"
+  properties:
+    cluster_id: UUID
+    cluster_name: string
+
+- name: cluster_deregister_failed
+  message: "Failed to deregister cluster. Error: {error}"
+  event_type: cluster
+  severity: "error"
+  properties:
+    cluster_id: UUID
+    error: string
+
+- name: deregistered_cluster
+  message: "Deregistered cluster: {cluster_id}"
+  event_type: cluster
+  severity: "info"
+  properties:
+    cluster_id: UUID
+
+- name: cluster_validation_falling
+  message: "Cluster validation '{validation_id}' that used to succeed is now failing"
+  event_type: cluster
+  severity: "warning"
+  properties:
+    cluster_id: UUID
+    validation_id: string
+
+- name: cluster_validation_fixed
+  message: "Cluster validation '{validation_id}' is now fixed"
+  event_type: cluster
+  severity: "info"
+  properties:
+    cluster_id: UUID
+    validation_id: string
```

Comment:
Add the validation message as a property. This could be really useful for debugging and not required to go into logs.